### ISSUE TITLE
Inbox adds extra linefeeds in Chrome, Edge

### ISSUE THREAD

**Issue by dwasifar** [roundcube/larry#3]
In Chrome and Edge, emails with long subject lines are getting an extraneous line break after the read status icon:

![image](https://github.com/roundcube/roundcubemail/assets/139193765/4672018b-9eed-46ad-9132-26bacefaa94b)

Shorter subject lines that don't require truncating to display are not affected.  This is only in the Larry skin; the Elastic skin is not affected.

Firefox is not affected:

![image](https://github.com/roundcube/roundcubemail/assets/139193765/721f8d83-2015-43b1-abd1-5996c3718712)

These screenshots are from Linux Mint Debian Edition, but the same behavior occurs in Windows.

Tested with Roundcubemail 1.6.2, Chrome 114.0.5735.198, Edge 114.0.1823.79, Firefox 115.0.

**Comment by alecpl** [comment]
I'm unable to reproduce, but my Chromium is older. Did you try with all plugins disabled?

**Comment by dwasifar** [comment]
Yes.  There were no active plugins during testing.  (I assume you mean Roundcube plugins and not Chrome/Edge plugins.)

I installed Chromium and tried that, and I am seeing the problem there too.  The version of Chromium that came in from the Mint repo is 114.0.5735.198; what version did you test with?

**Comment by dwasifar** [comment]
As an experiment, I installed the Gene Hawkins "Larry" variants (Autumn Larry, Blue Larry, Green Larry, etc) and the commercial Litecube-F skin from RoundcubePlus, and the problem exhibits in those as well.

Blue Larry:
![image](https://github.com/roundcube/larry/assets/139193765/67ae4afc-2fe2-4419-9ec9-2f74389cca8f)

Litecube-F:
![image](https://github.com/roundcube/larry/assets/139193765/8bb0a91a-3a80-4952-9418-ea8b9320c3a7)

**Comment by alecpl** [comment]
Fixed and released version 1.6.1 of the skin.

ps. It worked in Chrome 106.

**Comment by dwasifar** [comment]
Confirmed that fixes it.  Thanks, you're quick!

I also replicated the mail.css and mail.min.css change to one of the Hawkins variants, and it fixed that too.  (No luck with Litecube, though, it's too different.)

However, I notice the Settings page looks a tiny bit wonky now:

Old:
![image](https://github.com/roundcube/larry/assets/139193765/18c8b506-7962-4ef1-aa0d-3f3921b29e0d)

New:
![image](https://github.com/roundcube/larry/assets/139193765/ada4efef-dfe4-4613-a3d7-032284cbd795)

That can't be related to this change, can it?
